frontend/src/Components/ExternalHostBox/ExternalHostBox.tsx

```diff
@@ -0,0 +1,28 @@
+import { useTranslation } from 'react-i18next';
+import { Link, Text } from '~/Components';
+import { KEY } from '~/i18n/constants';
+import { COLORS } from '~/types';
+import styles from './ExternalHostBox.module.scss';
+
+type ExternalHostBoxProps = {
+  host: string;
+  host_link: string;
+};
+
+export function ExternalHostBox({ host, host_link }: ExternalHostBoxProps) {
+  const { t } = useTranslation();
+  return (
+    <div className={styles.external_host_box_wrapper}>
+      <Text>
+        {`${t(KEY.event_external_host_message)}: `}{' '}
```

Comment:
Unødvendig string interpolation

```suggestion
       {t(KEY.event_external_host_message)}: {' '}
```

Reply:
09d75e2a97fb0d951fad0054e05e1513b9508d8d

Reply:
09d75e2a97fb0d951fad0054e05e1513b9508d8d